docs/cmd/kn_service_create.md

```diff
@@ -56,13 +57,15 @@ kn service create NAME --image IMAGE [flags]
       --lock-to-digest           keep the running image for the service constant when not explicitly specifying the image. (--no-lock-to-digest pulls the image tag afresh with each new revision) (default true)
       --max-scale int            Maximal number of replicas.
       --min-scale int            Minimal number of replicas.
+      --mount stringArray        Mount a ConfigMap (prefix cm: or config-map:), a Secret (prefix secret: or sc:), or an existing Volume (without any prefix) on the specified directory. Example: --mount /mydir=cm:myconfigmap, --mount /mydir=secret:mysecret, or --mount /mydir=myvolume. When a configmap or a secret is specified, a corresponding volume is automatically generated. You can use this flag multiple times. To unmount, append "-" to the mounted directory, e.g. --volume /mydir-. When unmounting, the automatically generated volume is removed as well.
```

Comment:
In the last part, you mean `--mount /mydir`, not `--volume /mydir`, or ?

I would rephrase that last sentence to:

> For unmounting a directory append "-" e.g. --mount /mydir- which also removes any auto-generated volume.

(the last sentence is a bit of a simplication as it only removes the auto-generated volume for this mount, but I think this is clear enough).

Reply:
Oh, my mistake. I fixed it as your suggestion.